vtex-apps/store
# Add Order By Score (ASC and DESC)

#### What is the purpose of this pull request?

Use the product Search order by Score DESC and ASC. 
Updated name of message files to the new format (en.json required)

#### What problem is this solving?

Clients want order the Search Results by default with the product Score field.

#### How should this be manually tested?

1. In StoreFront go to Search Results Page.
2. Change the "Order By Selector"  to ordenation.score.ascending or ordenation.score.descending
3. Check the results are the same that the Search API:  https://{{acccount}}.vtexcommercestable.com.br/api/catalog_system/pub/products/search/{department}/{category}/?map=c,c&O=OrderByScoreDESC

#### Types of changes
